OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -19,6 +19,12 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Shape of actor for damage calculations.")]
 	public class HitShapeInfo : ConditionalTraitInfo
```

Comment:
`BodyOrientation` is used to translate between local and world coordinates, which is important even for stationary actors because that conversion might need camera angle or perspective fudges.